user_sync/config.py

```diff
@@ -423,15 +476,28 @@ def get_rule_options(self):
         }
         return result
 
-    def create_dashboard_options(self, connector_config_sources, owner):
-        connector_config = self.get_dict_from_sources(connector_config_sources, owner)
-        enterprise_section = connector_config.get('enterprise')
+    def create_dashboard_options(self, config_filename, additional_connector_config_sources, owner):
```

Comment:
Why did you feel the need to distinguish the position of the filename as opposed to letting any entry in the list be a filename and just act as another source?  That's the "YAML way".